owasp-dependency-check-suppressions.xml

```diff
@@ -806,4 +806,13 @@
     <!-- seems not applicable to plugin -->
     <cve>CVE-2022-45048</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+     file name: hadoop-client-runtime-3.3.5.jar
+     ]]></notes>
+    <!-- this one is windows only? https://nvd.nist.gov/vuln/detail/CVE-2022-26612 -->
```

Comment:
This one says it was addressed in 3.2.3. Looks like we're using 3.3.5. I would think it's irrelevant just on that basis.